packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -543,7 +558,38 @@ export class SolidParser {
         }
 
         // Split the file into lines
-        const lines = data.split("\n");
+        // Preprocess line data
+        const linesOBJ = data.split("\n");
+        const lineLines: string[][] = [];

```

Comment:
the three lines could look more expressive like this without the index need.:
```
const lineLines: string[][] = [];
let currentGroup: string[] = [];
lineLines.push(currentGroup);
```

Reply:
Changed.